sdk/core/azure-core-http-netty/src/test/java/com/azure/core/http/netty/NettyAsyncHttpClientTests.java

```diff
@@ -440,41 +436,6 @@ public void failedProxyAuthenticationReturnsCorrectError() {
         }
     }
 
-    @Test
```

Comment:
Any reason why these tests were removed?

Reply:
I was inherited from https://github.com/Azure/azure-sdk-for-java/pull/23123/files#diff-0d1d0c6bf45a1b59836e72d0353f7eb7e8ab4cd2c723cc897228e2bb039e640eL443

Reply:
We should revert both removals as they're still tests in main and cover some key scenarios